src/commands/build.ts

```diff
@@ -63,6 +64,8 @@ async function build_action(options: BuildOptions, context_file: string): Promis
       buildArgs.push(path.join(Deno.cwd(), context, build_options.context));
     }
 
+    console.log('Running: docker', ...buildArgs);
```

Comment:
Is this supposed to be here? It is possible a build arg would have sensitive information.

Reply:
Removed.